pages/docs/mutation.en-US.md

```diff
@@ -42,11 +42,10 @@ With `mutate`, you can update your local data programmatically, while
 revalidating and finally replace it with the latest data.
 
 ```jsx
-import useSWR, { useSWRConfig } from 'swr'
+import useSWR from 'swr'
 
 function Profile () {
-  const { mutate } = useSWRConfig()
-  const { data } = useSWR('/api/user', fetcher)
+  const { data, mutate } = useSWR('/api/user', fetcher)
```

Comment:
If you're using `boundMutate`, it doesn't take the 1st arg `key`.

`const { mutate } = useSWRConfig()` is for accessing the global mutate